source/Renderer/renderer.js

```diff
@@ -304,7 +304,7 @@ class gltfRenderer
         }
 
         // If any transmissive drawables are present, render all opaque and transparent drawables into a separate framebuffer.
-        if (this.transmissionDrawables.length > 0) {
+        if (this.transmissionDrawables.length > 0 || this.transparentDrawables.length > 0) {
```

Comment:
It is not necessary to apply the transmissive pass when no transmissive drawables are given. Probably, it is sufficient to apply line 322 ( _this.transparentDrawables = currentCamera.sortPrimitivesByDepth(state.gltf, this.transparentDrawables);_ ) in a separate if statement before drawing.